packages/server/test/hooks.spec.ts

```diff
@@ -574,7 +574,6 @@ describe('Hooks', () => {
                 expect(invocationErrorHook.error).not.toHaveBeenCalled();
                 expect(clientErrorHook.error).not.toHaveBeenCalled();
                 expect(globalErrorHook.error).not.toHaveBeenCalled();
-                done();
```

Comment:
This was wrong even in the server, but due to a race it wasn't causing a problem. It's confusing though.